django/contrib/admin/templates/admin/base.html

```diff
@@ -108,9 +108,9 @@
           <br class="clear">
         </div>
         <!-- END Content -->
-        {% block footer %}<div id="footer"></div>{% endblock %}
       </main>
     </div>
+    <footer id="footer">{% block footer %}{% endblock %}</footer>
```

Comment:
Can you add a small test? For example:
```diff
diff --git a/tests/admin_views/tests.py b/tests/admin_views/tests.py
index cb61c88941..429a79c062 100644
--- a/tests/admin_views/tests.py
+++ b/tests/admin_views/tests.py
@@ -1605,6 +1605,10 @@ class AdminViewBasicTest(AdminViewBasicTestCase):
             '<main id="content-start" class="content" tabindex="-1">',
         )
 
+    def test_footer(self):
+        response = self.client.get(reverse("admin:index"))
+        self.assertContains(response, '<footer id="footer">')
+
     def test_aria_describedby_for_add_and_change_links(self):
         response = self.client.get(reverse("admin:index"))
         tests = [

```

Reply:
Thanks. I added a test based on the existing header test!